templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -428,6 +428,7 @@
 	<input type="hidden" id="repolink" value="{{$.RepoRelPath}}">
 	<input type="hidden" id="repoId" value="{{.Repository.ID}}">
 	<input type="hidden" id="crossRepoSearch" value="{{.AllowCrossRepositoryDependencies}}">
+	<input type="hidden" id="type" value="{{.IssueType}}">
 	<!-- I know, there is probably a better way to do this -->
```

Comment:
Not relevant for this PR, but we should get rid of this comment or at least hide it from the rendered output:

```suggestion
	{{/* I know, there is probably a better way to do this */}}
```